### PR TITLE
DDCE-5157 : Toggle images + code cleanup

### PR DESCRIPTION
**Pull request by lmugnai**

### CODE REVIEW

**Review by EdCope — CHANGES_REQUESTED** [review]
Works and presume for this initial fix we don't need to cover [these end points](https://github.com/hmrc/binding-tariff-ruling-frontend/blob/509370e63f447aaa72599ae45cf908fee6404c56/conf/app.routes#L16-L17) as they are still accessible should you have the id.

This following acceptance test will need reviewing for the flag state.

https://github.com/hmrc/digital-tariffs-acceptance-tests/blob/f3c226a8aeedaabdae81a995c039ec258f44f14f/src/test/resources/features/rulings-ui/PublicRulingsSearch.feature#L32

File: test/unit/uk/gov/hmrc/bindingtariffrulingfrontend/views/RulingViewSpec.scala [hunk 1]
```diff
@@ -121,6 +127,23 @@ class RulingViewSpec extends ViewSpec {
           )
         }
 
+        "do not render images with correct href (images toggle off)" in {
+
+          given(appConfig.displayImages) willReturn false
```
[the remark this answers is not in the document]
Comment (lmugnai) [reply test/unit/uk/gov/hmrc/bindingtariffrulingfrontend/views/RulingViewSpec.scala:132]: Let's use given here for now as it's used a lot in this repo. If we want to change this let's have a ticket to refactor all.

File: app/uk/gov/hmrc/bindingtariffrulingfrontend/workers/RulingsWorker.scala [hunk 2]
```diff
@@ -49,14 +49,13 @@ class RulingsWorker @Inject() (
   implicit val ec: ExecutionContext = system.dispatchers.lookup("rulings-worker")
   implicit val hc: HeaderCarrier    = HeaderCarrier(extraHeaders = authHeaders(appConfig)(HeaderCarrier()))
 
-  val StreamPageSize                        = 1000
-  val StreamPagination: Pagination          = SimplePagination(pageSize = StreamPageSize)
-  val LocalDateFormatter: DateTimeFormatter = DateTimeFormatter.ISO_LOCAL_DATE
```
[the remark this answers is not in the document]
Comment (lmugnai) [reply app/uk/gov/hmrc/bindingtariffrulingfrontend/workers/RulingsWorker.scala:54]: Done

File: project/plugins.sbt [hunk 3]
```diff
@@ -13,7 +13,7 @@ addSbtPlugin("com.typesafe.play"   % "sbt-plugin"               % "2.8.20")
 addSbtPlugin("io.github.irundaia"  % "sbt-sassify"              % "1.5.2")
 addSbtPlugin("com.beautiful-scala" % "sbt-scalastyle"           % "1.5.1")
```
[the remark this answers is not in the document]
Comment (lmugnai) [reply project/plugins.sbt]: Done

**Review by EdCope — APPROVED** [review]

**Pull request merged**